### PR TITLE
Switched menu from discovery to search.

### PR DESCRIPTION
**Pull request by mickel1138**
The Drupal part still have to be done.

### CODE REVIEW

File: templates/Menus/nas-main.html.twig [hunk 1]
```diff
@@ -64,4 +64,4 @@
             </div>
         </li>
     </ul>
-</div>
+</div>
```
Comment (praneethpr) [inline templates/Menus/nas-main.html.twig:67]: Do you want to revert this change?

Comment (mickel1138) [reply templates/Menus/nas-main.html.twig:67]: Did I break something? Is it just a new line?

Comment (praneethpr) [reply templates/Menus/nas-main.html.twig:67]: Just the new line
